hack/generated/pkg/reconcilers/azure_deployment_reconciler.go

```diff
@@ -369,8 +369,6 @@ func (r *AzureDeploymentReconciler) DetermineCreateOrUpdateAction() (CreateOrUpd
 	}
 
 	if hasOngoingDeployment {
-		r.log.V(3).Info("Have existing deployment ID, will monitor it", "deploymentID", ongoingDeploymentID)
```

Comment:
If something stalls and never makes progress, this log line might be the key to spotting the situation. 
Suggest restoring this, but at Verbose level.

Reply:
I removed this because we actually already logged this above on line 355 pretty much, so this line was duplicate. Acutally the above line is better because it also includes the current `Ready` condition and `hasChanged`.